client/index.js

```diff
@@ -115,17 +109,18 @@ typeSelect.onchange = function () {
 
 addInteractions();
 
-/**
- * Read data from form and trigger WFS-T request
- */
-
+// Read data from form and trigger WFS-T request
 function sendData() {
   const elem = document.getElementById("attrs")
-  let properties = {};
+  var properties = {};
```

Comment:
var ? var is not used anymore, it was replaced by let and const

Reply:
I knew I should not use one of them, but I got them mixed up :)